subjects/devops/in-back-ground/README.md

```diff
@@ -4,11 +4,11 @@
 
 Create a script `in-back-ground.sh` which will execute a job in the background that will do the following:
 
-- Run the command `cat` on the file "facts" which will read the contents of the file and print it to `stdout`.
+- Run the command `cat` on the file [facts](../../../sh/tests/left/facts) which will read the contents of the file and print it to `stdout`.
```

Comment:
If ever the tests move, the link will be broken. No emergency, just a comment.

Reply:
got it, thanks!